WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -103,10 +111,38 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         coordinator?.start()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        configureFeaturedImage()
+
+        featuredImage.configure(scrollView: scrollView,
+                                navigationBar: navigationController?.navigationBar)
+
+        featuredImage.applyTransparentNavigationBarAppearance(to: navigationController?.navigationBar)
+
+        guard !featuredImage.isLoaded else {
+            return
+        }
+
+        // Load the image
+        featuredImage.load { [unowned self] in
+            self.hideLoading()
+        }
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+
+        featuredImage.restoreNavigationBarAppearance()
+    }
+
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
         ReaderTracker.shared.start(.readerPost)
+
+        // Reapply the appearance, this reset the navbar after presenting a view
+        featuredImage.applyTransparentNavigationBarAppearance(to: navigationController?.navigationBar)
```

Comment:
A few concerns regarding the transparent navigation bar:

* While I think it's beautiful, it's kind clashes with the rest of the app (AFAIK we don't have that anywhere else)
* The above is not a big deal, but I find a little bit weird that the navigation is translucid but the toolbar isn't.
* I can see that when we scroll the featured image gets behind it and stays there. I'd expect to see the scrolling content behind it
* In light mode when you scroll the status bar is white (while the icons are black). Shouldn't the status bar change to black too?

Reply:
The statusbar fix is done here: dc06d075787b37684b84e79984fb247bb3a68999

Reply:
It should! It was working perfectly last night, let me take another look.

Reply:
🤔 I'm not able to reproduce this on my end: https://d.pr/i/VG6nPi

Reply:
Ah I reproduced it on device. My fix is a bit involved so I'm going to make a new PR for it.

Reply:
Will be fixed in another PR.